blue_level/pizzabot.js

```diff
@@ -0,0 +1,54 @@
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
+
+const validateOrderName = (orderName) => {
```

Comment:
There is an arrow function convention which says that if we pass to our function only one argument, we can omit parentheses `()`, and rather write :

`const validateOrderName = orderName => { ... }`

than

`const validateOrderName = (orderName) => { ... }`

Important thing to notice - this convention should be applied ONLY when we pass one argument. If we would do this with multiple arguments, it would cause `SyntaxError` to occur

Reply:
This is a great tip. Thank you!